src/main/java/frc/robot/Constants.java

```diff
@@ -79,6 +79,8 @@ public AimModes previous() {
     }
   }
 
+  public static final boolean DebugMode = true;
```

Comment:
Should be false before we merge into main so we can tag it as competition-ready

Reply:
Fixed